docs/DatabaseCatalogAPI.yaml

```diff
@@ -261,11 +285,22 @@ components:
          type: string
         source_info:
           $ref: "#/components/schemas/SourceInfo"
-
+        redirects:
+          type: array
+          items:
+            description: >
+              The feed ID of a feed to use instead of the current one. 
```

Comment:
For clarity, I'd modify the description as: 

"The feed ID that should be used in replacement of the current one. A redirect is defined when several expired feeds are aggregated together or split into a new one."

Reply:
for the second sentence, I put 
`A redirect is defined when several expired feeds are aggregated together or split into new feeds.`